eyeshade/controllers/reports.js

```diff
@@ -481,6 +523,7 @@ module.exports.routes = [
   braveHapi.routes.async().path('/v1/reports/publishers/settlements').config(v1.publishers.settlements),
   braveHapi.routes.async().path('/v1/reports/publisher/{publisher}/statements').config(v1.publisher.statements),
   braveHapi.routes.async().path('/v1/reports/publishers/statements/{hash}').config(v1.publishers.statements),
+  braveHapi.routes.async().path('/v2/reports/publishers/statements/{settlementId}').config(v2.publisher.statements),
```

Comment:
v2.publisher.statements doesn't seem to fit the existing naming pattern

Reply:
for /v1, we use a BTC `{hash}` for /v2 we use a `{settlementId}` ...

Reply:
sure but it's plural `v1.publishers.statements` vs singular `v2.publisher.statements`